src/index.tsx

```diff
@@ -1,4 +1,4 @@
-import { Quill } from './components/Quill/Quill';
+import { DeltaStatic, Quill } from './components/Quill/Quill';
 import { providerRegistry } from './ProviderRegistry/index';
 
```

Comment:
```suggestion
export { DeltaStatic } from 'quill-delta';
```

Reply:
Dit zou je eventueel ook zo kunnen doen voor alle imports in `index.tsx`. Dus `import` vervangen door `export` en de `export` aan het einde verwijderen.